code/overview/cross-validation/leave-p-out.py

```diff
@@ -0,0 +1,47 @@
+# Example of LOOCV and LPOCV splitting
+
+import numpy
+from sklearn.model_selection import LeaveOneOut, LeavePOut
+
+# Configurable constants
+P_VAL = 2
+
+
+def print_result(method):
```

Comment:
The documentation looks good. I would suggest the same sort of changes that I mentioned in k-fold.py for consistency. That being said, the tab issues don't seem to present themselves with any of the reasonable P_VAL values so it might be fine to leave as is.